docs/tutorials/state_histograms.ipynb

```diff
@@ -91,27 +83,17 @@
-      "text/plain": "<Figure size 432x288 with 1 Axes>"
-     },
-     "metadata": {
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
-    }
-   ],
+   "outputs": [],
    "source": [
     "q = cirq.LineQubit.range(4)\n",
     "circuit = cirq.Circuit([cirq.H.on_each(*q), cirq.measure(*q)])\n",
     "result = cirq.Simulator().run(circuit, repetitions=100)\n",
-    "_ = cirq.vis.plot_state_histogram(result, plt.subplot())"
+    "_ = cirq.plot_state_histogram(result, plt.subplot())\n",
+    "plt.show()"
```

Comment:
I don't think `plt.show()` is needed in colab; is it needed in jupyter?

Reply:
You are right, it's not needed and colab and that's why the original notebook (also created by me) didn't contain it. But I was testing on jupyter this time and looks like it's needed there so I decided to add it.